sdk/storage/AzureStorageBlob/Source/Options/ClientMethodOptions.swift

```diff
@@ -432,13 +607,103 @@ public struct UploadBlobOptions: RequestOptions, Codable, Equatable {
         self.customerProvidedEncryptionKey = customerProvidedEncryptionKey
         self.customerProvidedEncryptionScope = customerProvidedEncryptionScope
         self.encoding = encoding
-        self.timeoutInSeconds = timeoutInSeconds
+        self.timeout = timeout
     }
 
     // TODO: Evalute whether serializing/deserializing dispatchQueue is necessary
     enum CodingKeys: CodingKey {
         case clientRequestId, cancellationToken, leaseAccessConditions, modifiedAccessConditions,
             encryptionOptions, customerProvidedEncryptionKey, customerProvidedEncryptionScope,
-            encoding, timeoutInSeconds
+            encoding, timeout
+    }
+}
+
+/// User-configurable options for the `BlobsOperations.getMetadata` operation.
+public struct GetBlobMetadataOptions: RequestOptions {
+    /// A client-generated, opaque value with 1KB character limit that is recorded in analytics logs.
+    public let clientRequestId: String?
+
+    /// A token used to make a best-effort attempt at canceling a request.
+    public let cancellationToken: CancellationToken?
+
+    /// A dispatch queue on which to call the completion handler. Defaults to `DispatchQueue.main`.
+    public var dispatchQueue: DispatchQueue?
+
+    /// A `PipelineContext` object to associate with the request.
+    public var context: PipelineContext?
+
+    /// An `Rfc1123Date` specifying the snapshot you wish to query.
+    public let snapshot: Rfc1123Date?
+
+    /// A n`Rfc1123Date` specifying the version of the blob to query.
+    public let versionId: Rfc1123Date?
+
+    /// The timeout parameter is expressed in seconds. This method may make
+    /// multiple calls to the Azure service and the timeout will apply to
+    /// each call individually.
+    public let timeout: TimeInterval?
+
+    /// Initialize an `UploadBlobOptions` structure.
+    /// - Parameters:
+    ///   - clientRequestId: A client-generated, opaque value with 1KB character limit that is recorded in analytics
+    ///     logs.
+    ///   - cancellationToken: A token used to make a best-effort attempt at canceling a request.
+    ///   - dispatchQueue: A dispatch queue on which to call the completion handler. Defaults to `DispatchQueue.main`.
+    ///   - timeout: The timeout parameter is expressed in seconds. This method may make multiple calls to the
+    ///     Azure service and the timeout will apply to each call individually.
+    public init(
```

Comment:
Missing some parameters in the doc comments

Reply:
This is why using Autorest is the way to go. :)  Fixed.